packages/compiler/src/lib/encoded-names.ts

```diff
@@ -39,7 +39,11 @@ export function $encodedName(
   existing.set(mimeType, name);
 }
 
-function getEncodedName(program: Program, target: Type, mimeType: string): string | undefined {
+export function getEncodedName(
```

Comment:
do you really need to export this? this add confusion with `resolveEncodedName` If you do please add TSDoc on it and explain the difference on both